File: tournaments/crossingSum/crossingSum.cs
int crossingSum(int[][] matrix, int a, int b) {
    var rows = matrix.Length;
    var cols = matrix[0].Length;
    var result = 0;
    
    for (int i = 0; cols > i; ++i) {
        result += matrix[a][i];
    }
    
    for (int i = 0; rows > i; ++i) {
        if (i != a) {
            result += matrix[i][b];
        }
    }
    
    return result;
}

File: tournaments/crossingSum/crossingSum.cpp
int crossingSum(std::vector<std::vector<int>> matrix,
                int row, int column) {

  int result = 0;
  for (int i = 0; i < matrix.size(); i++) {
    result += matrix[i][column];
  }
  for (int i = 0; i < matrix[0].size(); i++) {
    result += matrix[row][i];
  }
  result -= matrix[row][column];

  return result;
}
